src/Autofac.Extras.AttributeMetadata/Autofac.Extras.AttributeMetadata.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Autofac allows you to register metadata along with dependencies for additional resolution flexibility. This extension enables that metadata to be registered through attributes.</Description>
     <!-- VersionPrefix patched by AppVeyor -->
     <VersionPrefix>0.0.1</VersionPrefix>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Is there a benefit to adding `netstandard2.1` here? I see it adds a special updated reference below which otherwise wouldn't be required; and the Autofac.Mef package stuck at `netstandard2.0` support. If there's value (not just "because core Autofac does" but actually some technical benefit) I'm fine with it, but I wanted to check.

Reply:
Yeah you are right, didn't pay proper attention while doing so. Actually in one of the other packages I noticed that it doesn't compile on netstandard 2.1 without an additional Nuget package. Here it wasn't the case. Will fix the remaining stuff, also in the other PR, latest tommorow.

Reply:
Good to go now I think!